go/vt/srvtopo/resilient_server.go

```diff
@@ -215,7 +215,7 @@ func NewResilientServer(base *topo.Server, counterPrefix string) *ResilientServe
 		topoServer:   base,
 		cacheTTL:     *srvTopoCacheTTL,
 		cacheRefresh: *srvTopoCacheRefresh,
-		counts:       stats.NewCounters(counterPrefix + "Counts"),
+		counts:       stats.NewCountersWithLabels(counterPrefix+"Counts", "resilient srvtopo server operations", "type"),
```

Comment:
Another naming nitpick... descriptions should either all start with a capital letter or not :)